deploy/cluster_role.yaml

```diff
@@ -10,8 +10,6 @@ rules:
   verbs:
   - get
   - list
-  - patch
-  - update
```

Comment:
I noticed it seems like the oauth controller, like the clusterrole and group ones, seems to do an `Update` to add/remove finalizers from it [0] - it may potentially need some `oauth/finalizers` RBAC here, not sure.

[0] https://github.com/openshift/osd-metrics-exporter/blob/009a7aff944b4950bd59082792e298619988d01f/pkg/controller/oauth/oauth_controller.go#L97

Reply:
Nice catch, thanks! I'll add that permission.

I haven't tested all code paths yet so I expect a few more of these sorts of things to crop up.